.circleci/config.yml

```diff
@@ -780,11 +775,25 @@ jobs:
       - run:
           name: Build Wasm Release
           command: cargo build --release --target wasm32-unknown-unknown --locked
+      - run:
+          name: Install compile_module
+          # Uses --debug for performance/speed
+          # FIXME: Change when `compile_module` (part of `cosmwasm-0.14.0`) is published
+          #command: cargo install --debug --example compile_module -- cosmwasm-vm
+          command: cargo install --debug --git https://github.com/CosmWasm/cosmwasm --branch=contract-sanity-checking --example compile_module -- cosmwasm-vm
       - save_cache:
           paths:
             - /usr/local/cargo/registry
             - target
           key: cargocache-wasm-rust:1.50.0-{{ checksum "~/project/Cargo.lock" }}
+      - run:
+          name: Compile wasm files
+          command: |
+            for W in ./target/wasm32-unknown-unknown/release/*.wasm
+            do
+              echo -n "Compiling `basename $W`... "
+              compile_module $W || exit $?
```

Comment:
Nice line.

And I like where this is placed. Doesn't add much more delay to the CI and ensures compatibility

Reply:
Yes, this is optimal in that sense. Also compiling this utility for debug, which is twice as fast.